src/test/java/org/jnode/fs/xfs/XfsFileSystemTest.java

```diff
@@ -230,6 +230,10 @@ public void testLeafAttributes() throws Exception {
             XfsEntry entry = DataTestUtils.getDescendantData(new XfsFileSystemType().create(device, true), "leaf-attr.txt");
 
             INode leafAttributeINode = entry.getINode();
+
+            // Only v3 has this property.
+            assertThat(((INodeV3) leafAttributeINode).getCreated(), is(1643387236646L));
```

Comment:
Could apply `getDate()` to `getCreated()` to get a string representation of the date, then we could compare when reading the test data e.g. from linux.